Tests/AlgoliaSearch/ReindexCommandTest.php

```diff
@@ -83,7 +86,7 @@ public function reIndex($use_entity_alias, $safe = true)
 
         $input = new ArrayInput($options);
 
-        $this->command->run($input, new ConsoleOutput());
+        $this->command->run($input, new NullOutput());
```

Comment:
I don't think it's the same, I'm using Homestead and I think pretty much with everything by default.